atd-toolbox/fatalities_backfill/fatalities_backfill.sql

```diff
@@ -0,0 +1,13 @@
+-- This is a one time script to populate the fatalities table with historical data
+
+INSERT INTO fatalities (crash_id, primaryperson_id, location, year)
+SELECT atd_txdot_primaryperson.crash_id, primaryperson_id, CONCAT_WS(' ', rpt_block_num, rpt_street_pfx, rpt_street_name, '(', rpt_sec_block_num, rpt_sec_street_pfx, rpt_sec_street_name, ')'), TO_CHAR(crash_date,'yyyy')
+FROM atd_txdot_primaryperson
+JOIN atd_txdot_crashes ON atd_txdot_primaryperson.crash_id = atd_txdot_crashes.crash_id
+WHERE prsn_injry_sev_id = 4 AND ST_CONTAINS((SELECT geometry FROM atd_jurisdictions WHERE atd_jurisdictions.id = 5), position);
```

Comment:
the `ST_CONTAINS` checks if crash was in austin

Reply:
I didn't catch this in the triggers PR, but is there a reason to use `ST_CONTAINS` with this condition as opposed to the `austin_full_purpose` flag? Not being critical but want to know for my own understanding. 🙏